apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -88,7 +89,7 @@ export class AppLivechatBridge extends LivechatBridge {
 
 		let agentRoom;
 		if (agent?.id) {
-			const user = await Users.getAgentInfo(agent.id);
+			const user = await Users.getAgentInfo(agent.id, settings.get('Livechat_show_agent_email'));
```

Comment:
I think we can ignore the settings call here since we're not using the email's value

Reply:
hmmm, not really sure. The original model implementation run the setting for all cases, so I mimic it on here. We may need to revisit these places later to see if we still need this data there, but didn't want to miss any place and cause a random issue somewhere